bin/run.sh

```diff
@@ -32,4 +18,143 @@ if [[ ! -v OSTORLAB_PLATFORM ]]; then
   exit 1
 fi
 
-exec java -jar ${PLUGIN_JAR} --api-key $OSTORLAB_API_KEY --file-path $OSTORLAB_FILE_PATH --artifacts-dir $OSTORLAB_ARTIFACTS_DIR --platform $OSTORLAB_PLATFORM
+# Add --scan-profile argument
+PLATFORM=""
+if [[ "${OSTORLAB_PLATFORM,,}" == "android" ]]; then
+  PLATFORM="android-apk"
+elif [[ "${OSTORLAB_PLATFORM,,}" == "ios" ]]; then
+  PLATFORM="ios-ipa"
+elif [[ "${OSTORLAB_PLATFORM,,}" == "link" ]]; then
+  PLATFORM="link"
+else
+  echo "Unsupported platform: ${OSTORLAB_PLATFORM}"
```

Comment:
what if it's a domain ?
and we need to explain what are the supported platforms in the error.

Reply:
We support only links and it is explained in the Readme